integration-tests/http/__tests__/product-category/admin/product-category.spec.ts

```diff
@@ -1,40 +1,51 @@
 import { medusaIntegrationTestRunner } from "medusa-test-utils"
-import { createAdminUser } from "../../../helpers/create-admin-user"
+import {
+  adminHeaders,
+  createAdminUser,
+} from "../../../../helpers/create-admin-user"
 
 jest.setTimeout(50000)
 
-const env = { MEDUSA_FF_MEDUSA_V2: true }
-const adminHeaders = {
-  headers: { "x-medusa-access-token": "test_token" },
-}
-
 medusaIntegrationTestRunner({
-  env,
+  env: {},
```

Comment:
Suggestion : if there is no env anymore in the different files you remove it entirely